### DISCUSSION TITLE
Dynamic Module Federation with Angular

### DISCUSSION THREAD

**Discussion by yuliaNadia** [webpack/webpack#17684] [General]
Hello!
I'm trying to use Module Federation with Dynamic Modules loading. I have an Angular app (shell and micro frontend called event management). In shell assets I have module-federation.manifest.json:
```typescript
{
  "event-management": "http://localhost:4201"
}
``` 

I load it on main.ts in shell:
```typescript
import { setRemoteDefinitions } from '@nrwl/angular/mf';

fetch('/assets/module-federation.manifest.json')
  .then((res) =>res.json())
  .then((definitions) => setRemoteDefinitions(definitions))
  .then(() => import('./bootstrap').catch((err) => console.error(err)));
``` 

I also load remote modules in shell rout:
```typescript
export const appRoutes: Routes = [
  {
    path: '',
    component: NxWelcomeComponent,
  },
  {
    path: 'event-management',
    loadChildren: () =>
      loadRemoteModule('event-management', './Module').then((m) => m.EventManagementModule)
  },
];
``` 
The task is to deploy it on production. I try to do it locally first (build shell and event-management, http-server .\dist\apps)
I go to the domain and port after run http-server and shell is working, but there is an error with loading remoteEntry.mjs
**Uncaught (in promise): TypeError: Failed to fetch dynamically imported module: http://localhost:4201/remoteEntry.mjs**

Could anyone please help me with it? I can't understand how to load remoteEntry.mjs dynamically. I can change the URL in the manifest file, but it doesn't help

**Comment by nadeemramsing** [comment]
Try:
`http-server .\dist\apps --cors`
